### PR TITLE
Add release drafter

### LINKED ISSUE #46: Add release-drafter to the project
See:  https://github.com/release-drafter/release-drafter/
And: https://github.com/meilisearch/integration-guides/blob/main/guides/release-drafter.md

### PR DESCRIPTION
**Pull request by mdubus**
Closes #46

### CODE REVIEW

File: .github/release-draft-template.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+name-template: 'v$RESOLVED_VERSION 🌻'
+tag-template: 'v$RESOLVED_VERSION'
+exclude-labels:
+  - 'skip-changelog'
+version-resolver:
+  minor:
+    labels:
+      - 'breaking-change'
+      - 'enhancement'
```
Comment (curquiza) [inline .github/release-draft-template.yml:9]: Since your dashboard is not stable (`0.X.Y`), do NOT increase `X` for a new feature.

Comment (curquiza) [reply .github/release-draft-template.yml:9]:
```suggestion
```

**Review by curquiza — CHANGES_REQUESTED** [review]
Top moumoute

**Comment by mdubus** [comment]
bors merge

**Pull request merged**